### PR TITLE
PLAT-126323: Add UI tests for agate/Input

### PR DESCRIPTION
**Pull request by adrian-cocoara-lgp**
Enact-DCO-1.0-Signed-off-by: Adrian Cocoara adrian.cocoara@lgepartner.com

### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
 - Added ui tests for Input

### Links
[//]: # (Related issues, references)
PLAT-126323

### CODE REVIEW

File: tests/ui/specs/Input/Agate-Input-Page.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 'use strict';
 const {Page} = require('@enact/ui-test-utils/utils');
 
-class AgateInputPage extends Page {
+class SpotlightMultiplePage extends Page {
```
Comment (daniel-stoian-lgp) [inline tests/ui/specs/Input/Agate-Input-Page.js:4]: why was the name changed?

**Review by daniel-stoian-lgp — APPROVED** [review]
LGTM

File: tests/ui/specs/Input/Agate-Input-Page.js [hunk 2]
```diff
@@ -11,9 +11,24 @@ class AgateInputPage extends Page {
 		super.open('Agate-Input-View', urlExtra);
```
Comment (jeonghee27) [inline tests/ui/specs/Input/Agate-Input-Page.js:11]: You don't need to use'-' for Page file name.
In the PLAT-126330, there was not.
Agate-PanelPage.js is Better.
In fact, I don't think it's good to prefix of "Agate-".
"-Spec" and "-View" is mandatory to filter on ui-test-utils. but Page is not. just use camelcase.

**Review by jeonghee27 — APPROVED** [review]
LGEM except file name

File: tests/ui/specs/Input/Agate-Input-Page.js (hunk 2 above)
Comment (stanca-pop-lgp) [reply tests/ui/specs/Input/Agate-Input-Page.js:11]: I see. Thank you!

**Comment by adrian-cocoara-lgp** [comment]
Added some more unit tests for `Input`. 

cc: @jeonghee27, @daniel-stoian-lgp, @stanca-pop-lgp

File: tests/ui/specs/Input/Input-specs.js [hunk 3]
```diff
@@ -0,0 +1,48 @@
+const Page = require('./InputPage');
+
+describe('Input', function () {
+
+	it('should have focus on first item at start', function () {
```
Comment (MikyungKim) [inline tests/ui/specs/Input/Input-specs.js:5]:
```suggestion
	it('should have focus on first input at start', function () {
```

**Review by MikyungKim — APPROVED** [review]
LGTM

**Pull request merged**